zigzag/zigzag_test_log.py

```diff
@@ -870,7 +870,7 @@ def qtest_test_step_logs(self):
             qtest_test_step_log = swagger_client.AutomationTestStepLog()
 
             # set the status based on the previous status
-            previous_position = order - 1 if order is not 0 else 0
+            previous_position = order - 1 if order != 0 else 0
```

Comment:
Not a fan of `is not`?

Reply:
Flake8 isn't 🤷‍♂️ not sure why this was never caught before.